lib/msf/core/db_manager/http/servlet_helper.rb

```diff
@@ -13,21 +13,22 @@ def set_error_on_response(error)
   end
 
   def set_empty_response
-    [200,  '']
+    set_json_data_response(response: '')
   end
 
   def set_json_response(data, includes = nil, code = 200)
-    headers = {'Content-Type' => 'application/json'}
+    headers = { 'Content-Type' => 'application/json' }
     [code, headers, to_json(data, includes)]
   end
 
   def set_json_data_response(response:, includes: nil, code: 200)
-    data_response = {"data": response}
+    data_response = { "data": response }
     set_json_response(data_response, includes = includes, code = code)
   end
 
-  def set_json_error_response(response:, includes: nil, code:)
-    error_response = {"error": response}
+  def set_json_error_response(error:, includes: nil, code:)
+    print_error "Error handling request: #{error.message}", error
```

Comment:
Should we be printing error here?

Reply:
IMO, yes, or logging it (but probably both). One thing I noticed while doing this ticket is we swallow a lot of errors and they don't surface correctly. It made tracking down issues with the new format difficult. 

For example, we'll pop up an error like "Error creating host: <message>" but the actual error is a few levels deeper and you spend a lot of time trying to get there.

Reply:
I understand. This method is for creating an error response message so the printing of the error seems a bit misplaced. Would that be better suited alongside the call to this method? Instead of printing the error would it be better if that were a call to log the error since the web service will soon be run as a service rather than a program running on a console?

Reply:
Ahh, I see what you're saying. This is outside of the scope of the name of the method, even though it is convenient to put here. I do agree. Maybe we should have a wrapper method that both logs/prints the error and then calls `set_json_error_response`. I'll clean this up.